examples/database/litefs/nginx/nginx.conf

```diff
@@ -0,0 +1,22 @@
+http {
+    upstream primary {
+        server primary:8080;
+    }
+    upstream all {
+        server primary:8080;
+        server replica:8080;
+    }
+    server {
+        listen 80;
+        location / {
+            # Basically all writes should go to this primary,
+            # while reads go to everything else.
+            if ($request_method ~ "(PUT|POST|PATCH|DELETE)") {
+                proxy_pass http://primary;
+            }
+            proxy_pass http://all;
```

Comment:
Nginx is weird. I went down a rabbithole learning about `if`. Apparently it is imperative, but they tried to make it support some declarative cases and now they have a [monster](https://www.nginx.com/resources/wiki/start/topics/depth/ifisevil/).

I think what you have works, but it reads like it should proxy pass to `all` in all cases (if it was just evaluated imperatively top to bottom). Just thought I would share that observation.